doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -268,6 +268,8 @@ nRF5340 Audio
 * Updated:
 
   * Low latency configuration to be used as default setting for the nRF5340 Audio application.
+  * Migrated build system to support :ref:`zephyr:sysbuild`.
+    This means that the old Kconfig used to enable FOTA updates no longer exists, and the :ref: `zephyr:_application-file-suffixes` ``fota`` must be used instead.
```

Comment:
```suggestion
  * Migrated build system to support :ref:`configuration_system_overview_sysbuild`.
    This means that the old Kconfig used to enable FOTA updates no longer exists, and the :ref: `file suffix <app_build_file_suffixes>` ``fota`` must be used instead.
```
We would rather link to our own NCS docs, instead of Zephyr. What do you think, @greg-fer?

Reply:
Yes.